android/expoview/src/main/java/versioned/host/exp/exponent/modules/universal/notifications/ScopedExpoNotificationPresentationModule.kt

```diff
@@ -0,0 +1,133 @@
+package versioned.host.exp.exponent.modules.universal.notifications
+
+import android.content.Context
+import android.os.Bundle
+import android.os.ResultReceiver
+import expo.modules.core.Promise
+import expo.modules.notifications.notifications.NotificationSerializer
+import expo.modules.notifications.notifications.interfaces.NotificationTrigger
+import expo.modules.notifications.notifications.model.Notification
+import expo.modules.notifications.notifications.model.NotificationContent
+import expo.modules.notifications.notifications.model.NotificationRequest
+import expo.modules.notifications.notifications.presentation.ExpoNotificationPresentationModule
+import expo.modules.notifications.service.NotificationsService
+import host.exp.exponent.kernel.ExperienceKey
+import host.exp.exponent.notifications.ScopedNotificationsUtils
+import host.exp.exponent.notifications.model.ScopedNotificationRequest
+import java.util.*
+
+class ScopedExpoNotificationPresentationModule(
+  context: Context,
+  private val experienceKey: ExperienceKey
+) : ExpoNotificationPresentationModule(context) {
+  private val scopedNotificationsUtils: ScopedNotificationsUtils = ScopedNotificationsUtils(context)
+
+  override fun createNotificationRequest(
+    identifier: String,
+    content: NotificationContent,
+    trigger: NotificationTrigger?
```

Comment:
Based on the other files, looks like this might be able to be non-nullable

Reply:
The only place I could find that this is called it is called with null: `NotificationRequest request = createNotificationRequest(identifier, content, null);`.